graphql/src/execution/query.rs

```diff
@@ -152,12 +154,25 @@ impl Query {
             validate(schema.document(), &query.document, &GRAPHQL_VALIDATION_PLAN);
 
         if !validation_errors.is_empty() {
-            return Err(validation_errors
-                .into_iter()
-                .map(|e| {
-                    QueryExecutionError::ValidationError(e.locations.first().cloned(), e.message)
-                })
-                .collect());
+            if !ENV_VARS.graphql.silent_graphql_validations {
+                return Err(validation_errors
+                    .into_iter()
+                    .map(|e| {
+                        QueryExecutionError::ValidationError(
+                            e.locations.first().cloned(),
+                            e.message,
+                        )
+                    })
+                    .collect());
+            } else {
+                warn!(
+                  &logger,
+                  "GraphQL Validation failure";
+                  "query" => &query.query_text,
+                  "variables" => &query.variables_text,
+                  "errors" => validation_errors.iter().map(|e| e.message.clone()).collect::<Vec<_>>().join("\n")
```

Comment:
I believe the `.join("\n")` part would result in multi-line logs, which we are currently trying to avoid. Could we change the `"errors"` section to something different?

Maybe use an array formatting, like: `["first error", "second error"]`?

Reply:
Sure, that makes sense. I changed it, thanks!